hub/core/chunk_engine/tests/common.py

```diff
@@ -71,6 +73,10 @@ def assert_chunk_sizes(
                 j,
             )
 
+            if chunk_name not in actual_chunk_names:
```

Comment:
looks like a dict. could a dict work here instead of the set/array combo?

Reply:
yeah that would be better